scripts/install-prereqs

```diff
@@ -14,7 +14,7 @@
 PACKAGES=('cmake' 'make' 'pkg-config')
 
 # Needed for Open Enclave scripts
-PACKAGES+=('clang-format-3.8' 'shellcheck')
+PACKAGES+=('clang-format-3.9' 'shellcheck')
```

Comment:
Can you check with @andschwa on this? I believe we pinned it to v3.8 because it needs to be consistent with what the CI pipeline uses.

Reply:
Yeah this shouldn't change until all of #728 is resolved because #711.